contrib/editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
```

Comment:
We do use tab for shell script though :angel:, I think it's good as is. `*.md` is handled down :stuck_out_tongue_closed_eyes: